aea/helpers/pipe.py

```diff
@@ -511,8 +511,13 @@ def __init__(  # pylint: disable=unused-argument
         """
         self.logger = logger
         self._loop = loop
-
-        self._port = int(in_path)
+        if len(in_path.split(":")) == 1:
+            self._port = int(in_path)
+            self._host = "127.0.0.1"
```

Comment:
(minor) this can be a constant somewhere, or:

```
socket.gethostbyname(socket.getfqdn())
```

Reply:
blocking operation, can take a lot of time